src/sentry/tasks/reports.py

```diff
@@ -270,16 +319,48 @@ def prepare_project_issue_summaries(interval, project):
     return [new_issue_count, reopened_issue_count, existing_issue_count]
 
 
-def prepare_project_usage_summary(start__stop, project):
+def prepare_project_usage_outcomes(start__stop, project):
     start, stop = start__stop
-    return (
-        tsdb.get_sums(
-            tsdb.models.project_total_blacklisted, [project.id], start, stop, rollup=60 * 60 * 24
-        )[project.id],
-        tsdb.get_sums(
-            tsdb.models.project_total_rejected, [project.id], start, stop, rollup=60 * 60 * 24
-        )[project.id],
+
+    # XXX(epurkhiser): Tsdb used to use day buckets, where the end would
+    # represent a whole day. Snuba queries more accurately thus we must
+    # capture the entire last day
+    end = stop + timedelta(days=1)
+
+    query = Query(
+        dataset=Dataset.Outcomes.value,
+        match=Entity("outcomes"),
+        select=[
+            Column("outcome"),
+            Column("category"),
+            Function("sum", [Column("quantity")], "total"),
+        ],
+        where=[
+            Condition(Column("timestamp"), Op.GTE, start),
+            Condition(Column("timestamp"), Op.LT, end),
+            Condition(Column("project_id"), Op.EQ, project.id),
+            Condition(Column("org_id"), Op.EQ, project.organization_id),
+        ],
+        groupby=[Column("outcome"), Column("category")],
+        granularity=Granularity(ONE_DAY),
     )
+    data = raw_snql_query(query, referrer="reports.outcomes")["data"]
+
+    return {
+        "accepted_transactions": sum(
+            row["total"]
+            for row in data
+            if row["category"] == DataCategory.TRANSACTION and row["outcome"] == Outcome.ACCEPTED
```

Comment:
I would add a Condition for `outcome IN [ACCEPTED, FILTERED, RATE_LIMITED]` to the main query unless those are the only 3 options.

Reply:
👍 good call